TESTS/mbed_platform/FileHandle/main.cpp

```diff
@@ -353,7 +375,17 @@ void test_fprintf_fscanf()
     TEST_ASSERT_TRUE(TestFile<FS>::functionCalled(TestFile<FS>::fnWrite));
     TEST_ASSERT_TRUE(std::ferror(file) != 0);
     TEST_ASSERT_TRUE(fprintf_ret < 0);
-
+#else
+    // Writing remaining available file space of 2 characters
+    // to make the further fscanf() test to pass.
```

Comment:
to make the further fscanf() test to pass --> to make further fscanf() test pass.